pkgs/development/libraries/snap7/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "snap7";
+  version = "1.4.0";
```

Comment:
there is a 1.4.2 version available when clicking "download" on the homepage. (on sourceforge)

Reply:
My bad, I will change it

Reply:
fixed